setup.cfg

```diff
@@ -1,12 +1,2 @@
 [wheel]
 universal = 1
-
-[pytest]
-python_files = test*.py
-addopts = --tb=native -p no:doctest
-norecursedirs = bin dist docs htmlcov script hooks node_modules .* {args}
-
-[flake8]
-ignore = F999,E501,E128,E124,E402,W503,E731,C901
-max-line-length = 100
-exclude = .tox,.git,*/migrations/*,node_modules/*,docs/*
```

Comment:
Is @se-bastiaan interested in adding lints or something? If so we shouldn't be deleting this I think.

Reply:
The `flake8` config is now kept (with some small changes). I also fixed all the pep8 violations.

Reply:
I copied basically everything from the original repo, this is one of them :p
However, that doesn't mean that I think lints are bad.